src/components/ContactDetails/ContactDetailsProperty.vue

```diff
@@ -374,6 +378,20 @@ export default {
 		},
 	},
 
+	created() {
+		this.bus.$on('focus-prop', (id) => {
+			if (id === this.propName) {
+				this.$nextTick(() => {
+					if (this.$refs.component.$el.querySelector('input') === undefined) {
+						console.warn('no input to focus found')
+					} else {
+						this.$refs.component.$el.querySelector('input').focus()
+					}
+				})
+			}
+		})
```

Comment:
It's probably a good idea to move the logic to a method and call `this.bus.$off('focus-prop', this.onFocusProp)` in `destroyed()` to prevent leaks.

Reply:
Didn't it focus at all or just not on adding a new prop? The focusing is just implemented for the case that the prop already exists and it's not a multiple one. (Going to add this for the add case)